xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.h

```diff
@@ -50,6 +50,8 @@ class CRenderManager;
 class IRenderMsg
 {
   friend CRenderManager;
+public:
+  virtual ~IRenderMsg() = default;
```

Comment:
Well, maybe it is actually a good idea (if not needed) to add a default virtual ctor, even to a c++ interface, as the compiler may generate a non-virtual dtor for that class:

<pre>
class IBatch
{
    public:
       virtual void f() = 0;
};

class A : public IBatch
{
    public:
       void f() {}
       ~A() {}
};

IBatch * a = new A();
a->f();    // calls A::f()
delete a;  // calls IBatch::~IBatch() not A::~A()
</pre> 

Source: https://stackoverflow.com/questions/2691611/destructors-for-c-interface-like-classes